### PR TITLE
Reduce ceph setup time

### PR DESCRIPTION
**Pull request by hkumarmk**
ceph::auth to be called after mon_config

This will reduce the time required for ceph setup.

### CODE REVIEW

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by bodepd** [comment]
please test this

**Comment by bodepd** [comment]
please test this

**Comment by bodepd** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by bodepd** [comment]
please test this

**Comment by bodepd** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

File: hiera/data/role/httpproxy.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@ ntp::servers:
 rjil::system::proxies:
   "no":
     url: "127.0.0.1,169.254.169.254,localhost,consul,jiocloud.com"
+
+resolvconf::nameservers: ['8.8.4.4','8.8.8.8']
```
Comment (bodepd) [inline hiera/data/role/httpproxy.yaml]: won't this break consul dns?

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by bodepd** [comment]
merged as #486

**Pull request closed without merging**